kratos/tests/geometries/test_triangle_2d_3.cpp

```diff
@@ -183,6 +196,28 @@ namespace Testing {
     KRATOS_CHECK(geom->IsInside(PointInEdge, LocalCoords, EPSILON));
   }
 
+  /** Checks the point local coordinates for a given point respect to the
+   * triangle. The baricentre of the triangle is selected due to its known
+   * solution.
+   */
+  KRATOS_TEST_CASE_IN_SUITE(Triangle2D3PointLocalCoordinates, KratosCoreGeometriesFastSuite) {
+
+    auto geom = GenerateIrregularTriangle2D3<Node<3>>();
+
+    // Compute the global coordinates of the baricentre
+    array_1d<double, 3> baricentre = ZeroVector(3);
+    const Geometry<Node<3>>::PointsArrayType geom_pts = geom->Points();
+    baricentre(0) = (geom_pts[0].X() + geom_pts[1].X() + geom_pts[2].X()) / 3.0;
+    baricentre(1) = (geom_pts[0].Y() + geom_pts[1].Y() + geom_pts[2].Y()) / 3.0;
+
+    // Compute the baricentre local coordinates
+    array_1d<double, 3> baricentre_local_coords = geom->PointLocalCoordinates(baricentre_local_coords, baricentre);
+
+    KRATOS_CHECK_NEAR(baricentre_local_coords(0), 1.0/3.0, 1e-5);
```

Comment:
1e-5 is not too high? The result is so approximated?

Reply:
I'm changing it. It is the analytical solution of the system, so there shouldn't be any problem.